src/pkg/blob/manager.go

```diff
@@ -129,6 +132,10 @@ func (m *manager) Delete(ctx context.Context, id int64) error {
 	return m.dao.DeleteBlob(ctx, id)
 }
 
+func (m *manager) UselessBlobs(ctx context.Context, timeWindowHours int64) ([]*models.Blob, error) {
+	return m.dao.GetBlobsNotRefedByProjectBlob(ctx, timeWindowHours)
```

Comment:
I think it would be a better design if timeWindowHousrs is handled at the manager level.
And let the func in `dao` pkg only cares about the timestamp where the update time should be no later than.

Reply:
It needs to be set by GC job, and let GC job to give a value. And for debugging, it can be set to 0.

Reply:
and why not leveage Pg's buildin func `interval` to handle it